ldk/csharp/OliveHelpsLDK/OliveHelpsLDK/Whispers/List.cs

```diff
@@ -23,22 +23,30 @@ public enum Align
 
     public class ListPair : ListBase
     {
-        public string Label { get; }
-        public string Value { get; }
-        public Style Style { get; }
-        public bool Copyable { get; }
+        public string Label { get; set; }
+        public string Value { get; set; }
+        public Style Style { get; set; }
+        public bool Copyable { get; set; }
     }
 
     public class ListMessage : ListBase
     {
-        public string Header { get; }
-        public string Body { get; }
-        public Style Style { get; }
-        public Align Align { get; }
+        public string Header { get; set; }
+        public string Body { get; set; }
+        public Style Style { get; set; }
```

Comment:
As we discussed, these never should have worked, as you can't really mutate a string through a getter like you could a List or Dictionary